packages/indicators/src/types.ts

```diff
@@ -22,7 +22,7 @@ export interface AnalyticCluster {
   dataValues: Record<Analytic['dataElement'], Analytic['value']>;
 }
 
-type TypeFields = Record<string, string | number | {}>;
+type TypeFields = Record<string, string | number | Record<string, unknown>>;
```

Comment:
After a looking at a few examples of this change, I prefer the old way...

Reply:
I preferred the old way too (in terms of syntax), but it's actually inaccurate...

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/ban-types.md#default-options

> Avoid the Object and {} types, as they mean "any non-nullish value".
This is a point of confusion for many developers, who think it means "any object type".

Reply:
Oh good to know, I definitely thought it meant any object type so it's worth the change